src/Evenement/EventEmitterTrait.php

```diff
@@ -68,10 +68,10 @@ public function once($event, callable $listener)
     public function off($event, callable $listener = null)
     {
         if ($listener !== null) {
-            return $this->removeListener($event, $listener);
+            $this->removeListener($event, $listener);
```

Comment:
Since your PR changes the flow of this function, can you append your commit with: 

```suggestion
            $this->removeListener($event, $listener);
            return;
```